src/UI/Component/Panel/Listing/Appointement.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/* Copyright (c) 2016 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Panel\Listing;
+
+/**
+ * Interface Appointment
+ * @package ILIAS\UI\Component\Panel\Listing
+ */
+interface Appointment extends \ILIAS\UI\Component\Component {
```

Comment:
I was also confused with the names, I expected only one appointment. Another option could be Panel/Listing/AppointmentList instead of Panel/Listing/Appointement.php and /PanelListing/Appointment instead of /Panel/Listing/AppointementItem.php